<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>1024</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>User Profile Explorer</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Selection Mode</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="source_mode_radio">
          <property name="text">
           <string>Source</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="target_mode_radio">
          <property name="text">
           <string>Target</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="feature_selector">
     <item row="1" column="1">
      <widget class="QComboBox" name="feature_combo_box">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string/>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Feature</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="scatter_chart">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>500</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="user_name_input">
       <property name="placeholderText">
        <string>@TwitterUserName</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="search_user_button">
       <property name="text">
        <string>Search </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Profile Distance:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="profile_distance_output">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>nan</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="tweet_display">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTextBrowser" name="source_user_tweet_output">
        <property name="placeholderText">
         <string>Source user representative data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="target_user_tweet_output">
        <property name="placeholderText">
         <string>Target user representative data</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
